honeycomb-core/src/twomap.rs

```diff
@@ -294,15 +295,15 @@ impl<const N_MARKS: usize> TwoMap<N_MARKS> {
     /// See [TwoMap] example.
     ///
     pub fn new(n_darts: usize, n_vertices: usize) -> Self {
-        let vertices = vec![[0.0, 0.0]; n_vertices];
+        let vertices = vec![Vertex2::from([0.0, 0.0]); n_vertices];
         let betas = vec![[0; TWO_MAP_BETA]; n_darts + 1];
 
         Self {
             vertices,
-            free_vertices: Vec::with_capacity(n_vertices),
+            unused_vertices: BTreeSet::new(),
```

Comment:
You loose the pre-allocation here, is it important?

Reply:
I will run benchmarks and compare 0.1.1 to 7fafa42 (which I plan on making 0.1.2 with) . I will add relevant comparisons somewhere, not sure where yet.